src/denostore.ts

```diff
@@ -29,23 +31,38 @@ export default class Denostore {
       route = '/graphql',
       defaultEx = undefined,
     } = args;
+
+    this.setSchemaProperty(schema);
     this.#usePlayground = usePlayground;
     this.#redisClient = redisClient;
-    this.#schema = schema;
     this.#router = new Router();
     this.#route = route;
     this.#defaultEx = defaultEx;
   }
 
+  setSchemaProperty(schema: GraphQLSchema | ExecutableSchemaArgs) {
```

Comment:
Very small but I think we want a return type of void here

Reply:
Updated it in the new commit